machinery-helper/mountpoints/mountpoints.go

```diff
@@ -29,53 +29,54 @@ var ProcMountsPath = "/proc/mounts"
 
 var remoteFileSystems = []string{"autofs", "cifs", "nfs", "nfs4"}
 var specialFileSystems = []string{"proc", "sysfs", "devtmpfs", "tmpfs", "rpc_pipefs", "fuse.gvfs-fuse-daemon"}
-var localFileSystems  = []string{"ext2", "ext3", "ext4", "reiserfs", "btrfs", "vfat", "xfs", "jfs"}
+var localFileSystems = []string{"ext2", "ext3", "ext4", "reiserfs", "btrfs", "vfat", "xfs", "jfs"}
 
-func parseMounts() map[string]string {
-  mount, _ := ioutil.ReadFile(ProcMountsPath)
-  mounts := make(map[string]string)
+// Returns the existing mountpoints in a system.
```

Comment:
comment on exported function ParseMounts should be of the form "ParseMounts ..."